src/PhraseApp.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp;
+
+use nediam\PhraseApp\PhraseAppClient;
+use Symfony\Component\Translation\MessageCatalogueInterface;
+use Translation\Common\Exception\StorageException;
+use Translation\Common\Model\Message;
+use Translation\Common\Storage;
+use Translation\Common\TransferableStorage;
+use Translation\PlatformAdapter\PhraseApp\Bridge\Symfony\XliffConverter;
+
+class PhraseApp implements Storage, TransferableStorage
+{
+    /**
+     * @var PhraseAppClient
+     */
+    private $client;
+
+    /**
+     * @var string
+     */
+    private $projectId;
+
+    /**
+     * @var array
+     */
+    private $localeToIdMapping;
+
+    /**
+     * PhraseApp constructor.
+     *
+     * @param PhraseAppClient $client
+     * @param string $projectId
+     * @param string $acountId
+     */
+    public function __construct(PhraseAppClient $client, string $projectId, array $localeToIdMapping)
+    {
+        $this->client = $client;
+        $this->projectId = $projectId;
+        $this->localeToIdMapping = $localeToIdMapping;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function get($locale, $domain, $key)
+    {
+        $translations = $this->client->request('translation.indexLocale', [
+            'project_id' => $this->projectId,
+            'locale_id' => $this->getLocaleId($locale),
+        ]);
+
+        foreach ($translations as $translation) {
+            if ($translation['key']['name'] === "$domain::$key") {
+                return new Message($key, $domain, $locale, substr($translation['content'], strlen($domain) + 2));
+            }
+        }
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
The interface description says: "Create a new translation or asset. If a translation already exist this function will do nothing." 

Is this really the case?